pkg/task/obtenant_flow.go

```diff
@@ -0,0 +1,115 @@
+package task
+
+import (
+	"github.com/oceanbase/ob-operator/api/v1alpha1"
+	tenantstatus "github.com/oceanbase/ob-operator/pkg/const/status/tenantstatus"
+	flowname "github.com/oceanbase/ob-operator/pkg/task/const/flow/name"
+	taskname "github.com/oceanbase/ob-operator/pkg/task/const/task/name"
+	"github.com/oceanbase/ob-operator/pkg/task/fail"
+)
+
+func CreateTenant() *TaskFlow {
+	return &TaskFlow{
+		OperationContext: &v1alpha1.OperationContext{
+			Name:         flowname.CreateTenant,
+			Tasks:        []string{taskname.CheckTenant, taskname.CheckPoolAndUnitConfig,taskname.CreateTenant},
```

Comment:
does create tenant do all the works to create a tenant, the two previous tasks are all doing some check

Reply:
Unit and pool have not been split yet because they are one-to-one and somewhat coupled in implementation. 
We can decouple them in future commit.

Reply:
Create tenant split into create unit and pool and create tenant now